VAMobile/src/components/WhatsNew.tsx

```diff
@@ -97,31 +92,25 @@ export const WhatsNew = () => {
 
   const bullets = getBullets() || []
 
-  const props: CollapsibleAlertProps = {
-    border: 'informational',
-    headerText: t('whatsNew.title'),
-    body: (
-      <>
-        <Box my={theme.dimensions.standardMarginBetween}>
-          <TextView accessibilityLabel={bodyA11yLabel}>{body}</TextView>
-          {bullets.length ? (
-            <Box mt={theme.dimensions.standardMarginBetween}>
-              <VABulletList listOfText={bullets} />
-            </Box>
-          ) : undefined}
-        </Box>
-        <Button onPress={onPress} label={t('whatsNew.dismissMessage')} buttonType={ButtonVariants.Secondary} />
-      </>
-    ),
-    a11yLabel: t('whatsNew.title'),
-    onExpand: expandCollapsible,
-    onCollapse: closeCollapsible,
-  }
-
   if (displayWN) {
     return (
       <Box mb={theme.dimensions.standardMarginBetween}>
-        <CollapsibleAlert {...props} />
+        <AlertWithHaptics
+          variant="info"
+          expandable={true}
+          header={t('whatsNew.title')}
+          headerA11yLabel={t('whatsNew.title')}
+          analytics={{ onExpand: expandCollapsible, onCollapse: closeCollapsible }}>
+          <Box my={theme.dimensions.standardMarginBetween}>
+            <TextView accessibilityLabel={bodyA11yLabel}>{body}</TextView>
+            {bullets.length ? (
+              <Box mt={theme.dimensions.standardMarginBetween}>
+                <VABulletList listOfText={bullets} />
+              </Box>
+            ) : undefined}
+          </Box>
+          <Button onPress={onPress} label={t('whatsNew.dismissMessage')} buttonType={ButtonVariants.Secondary} />
```

Comment:
Could potentially be simplified moving this to the `secondaryButton` prop to AlertWithHaptics.